test/pseudonymsys_ec_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func TestPseudonymsysEC(t *testing.T) {
 	dlog := dlog.NewECDLog(dlog.P256)
-	caClient, err := client.NewPseudonymsysCAClientEC(testGrpcServerEndpoint)
+	caClient, err := client.NewPseudonymsysCAClientEC(testGrpcClientConn)
```

Comment:
Oh, I didn't realize the `TestMain` is shared for all the tests in the package. All good then, thanks for teaching me this.